### DISCUSSION TITLE
Deploying nextjs project to docker

### DISCUSSION THREAD

**Discussion by kamolovd** [vercel/next.js#67659] [Help]
### Summary

after creating the build and image on Docker, the size of the image is 660 mb, since I know the nextjs build of a project that has SSR does not work without the node_modules folder, the size of which inside the docker image is 589 mb. How to reduce the size of node_modules?

**Comment by BorhanSaflo** [comment]
Consider switching to [pnpm](https://pnpm.io/) to reduce the size of your `node_modules`. pnpm can decrease the size of `node_modules` by using hard links and symbolic links to a global store. Here's a [comparison](https://pnpm.io/pnpm-vs-npm) between pnpm and npm.
